.github/workflows/release_linux.yml

```diff
@@ -37,16 +37,22 @@ jobs:
     steps:
     - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       with:
-        persist-credentials: false
+        persist-credentials: true
         submodules: true
 
+    - name: Configure Git safe directory
+      run: |
+        git config --global --add safe.directory /github/workspace
+        echo "SOURCE_DATE_EPOCH=$(git log -1 --pretty=%ct)" >> $GITHUB_ENV
```

Comment:
Looks like this is not "Configure Git safe directory" so I would move it into its separate step.

Reply:
fixed